src/Npgsql/NpgsqlConnector.cs

```diff
@@ -786,8 +786,7 @@ async Task ConnectAsync(NpgsqlTimeout timeout, CancellationToken cancellationTok
                     // cancellation and timeout. On older TFMs, we fake-cancel the operation, i.e. stop waiting
                     // and raise the exception, but the actual connection task is left running.
 
-// TODO: NET5_0 is here because of https://github.com/dotnet/sdk/issues/13377, remove for 5.0.0-rc2
-#if (NET461 || NETSTANDARD2_0 || NETSTANDARD2_1 || NETCOREAPP3_1) && !NET5_0
+#if !NET // i.e. >= NET5_0
```

Comment:
Uh, why not inverse the logic, since we always have an #else clause? So:

```c#
#if NET // i.e. >= NET5_0
// new stuff
#else
// old stuff
#endif

I'd also remove the comment. I know this thing is new, but it's up to us to know what it means etc.

Reply:
Because of bangs! We love them (:

Reply:
Oups, it should be:
```cs
#if !NET // i.e. < NET5_0
```
Do I make another PR to fix that ? or maybe just inverse the logic as proposed by @roji ?

Reply:
As @YohDeadfall pointed out to me several times, we already have enough bangs in the codebase thanks to nullable references 🤣 

Yeah, let's just inverse the logic (and maybe kill the comment altogether please??)

Reply:
Personally I see nothing bad how you did it, because there is still a conditional compilation. And yeah, I would like to keep the comment. It helps a lot since it's hard to imagine how it could be not .NET.